utils/ocp_cli.py

```diff
@@ -10,19 +10,21 @@ def __init__(self, provider):
 
         provider_cfme_data = provider.get_yaml_data()
         self.hostname = provider_cfme_data['hostname']
-        creds = conf.configuration.yaycl_config.credentials
-        if hasattr(creds, provider.key):
-            prov_creds = getattr(creds, provider.key)
-            self.username = prov_creds.username
-            self.password = prov_creds.password
-            self.ssh_client = SSHClient(hostname=self.hostname,
-                                        username=self.username,
-                                        password=self.password)
+        creds = provider_cfme_data.get('ssh_creds', None)
+
+        if not creds:
+            raise Exception('Could not find ssh_creds in provider\'s cfme data.')
+        if isinstance(creds, dict):
+            self.username = creds.get('username', None)
+            self.password = creds.get('password', None)
         else:
-            # Try with known hosts
-            self.ssh_client = SSHClient()
-            self.ssh_client.load_system_host_keys()
-            self.ssh_client.connect(self.hostname)
+            self.username = credentials[creds].get('username', None)
+            self.password = credentials[creds].get('password', None)
+
+        self.ssh_client = SSHClient()
```

Comment:
@mfalesni if possible, I really prefer to keep it as it is since this is currently work well in Jenkins and PRT and this PR include very  urgent issues. hope that this is not critical to use With.